File: DuDoong-Api/src/main/java/band/gosrock/api/ticketItem/dto/request/ApplyTicketOptionRequest.java
package band.gosrock.api.ticketItem.dto.request;


import io.swagger.v3.oas.annotations.media.Schema;
import javax.validation.constraints.Positive;
import lombok.Getter;
import lombok.RequiredArgsConstructor;

@Getter
@RequiredArgsConstructor
public class ApplyTicketOptionRequest {

    @Schema(nullable = false, example = "1")
    @Positive(message = "올바른 옵션그룹 고유 아이디를 입력해주세요")
    private Long optionGroupId;
}

File: DuDoong-Api/src/main/kotlin/band/gosrock/api/ticketItem/dto/request/ApplyTicketOptionRequest.kt
package band.gosrock.api.ticketItem.dto.request

import io.swagger.v3.oas.annotations.media.Schema
import javax.validation.constraints.Positive

data class ApplyTicketOptionRequest(
    @field:Schema(nullable = false, example = "1")
    @field:Positive(message = "올바른 옵션그룹 고유 아이디를 입력해주세요")
    val optionGroupId: Long? = null,
)
